main/uart_nic.cpp

```diff
@@ -47,18 +47,16 @@ extern "C" {
 }
 
 #include "uart0_driver.h"
+#include "esp_protocol/messages.hpp"
 
 // Externals with no header
 extern "C" {
     int ieee80211_output_pbuf(esp_aio_t *aio);
     esp_err_t mac_init(void);
 }
 
-#define FW_VERSION 12
-
-#define SCAN_MAX_STORED_SSIDS 64
-#define SSID_LEN 32
-#define BSSID_LEN 6
+static constexpr size_t SCAN_MAX_STORED_SSIDS = 64;
```

Comment:
When implementing this feature I took the common files as future library that would be shared between printers <-> esps, not directly between esps since I would like to merge both firmwares into one. This constant is not used in printer firmware right now, we send the message even if the index is out of range, and respond with invalid/empty message.